beta/kustomize/overlays/letsencrypt/README.md

```diff
@@ -0,0 +1,5 @@
+# Let's Encrypt
+
+To use the Let's Encrypt overlay, modify the value of the patch for `OP_TLS_DOMAIN`
```

Comment:
Same concern as the other comment on this around the phrasing:
I wonder if the phrasing here might be confusing since at this point, someone may not have created the A record since they'd not have deployed anything to the cluster and may not have an external IP?

Reply:
Resolved via main readme changes